microsoft-edge/webview2/how-to/remote-debugging-xbox.md

```diff
@@ -9,6 +9,7 @@ ms.subservice: webview
 ms.date: 10/17/2023
 ---
 # Remote debugging Xbox WebView2 WinUI 2 (UWP) apps
+<!-- todo: description: YAML field: remove "by using Remote Tools for Microsoft Edge"? -->
```

Comment:
in the description: YAML field, remove "by using Remote Tools for Microsoft Edge"?

rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/remote-debugging-xbox?branch=pr-en-us-3161

@champnic

Reply:
removed